app/services/station_service.rb

```diff
@@ -0,0 +1,11 @@
+class StationService < BaseService
+  def self.find_station(location)
+
+    response = conn('https://developer.nrel.gov').get("/api/alt-fuel-stations/v1/nearest.json") do |f|
+      f.params['location'] = location
+      f.params['api_key'] = ENV['station_key']
+    end
+    json = format_json(response)
+    json[:fuel_stations]
+  end
```

Comment:
Syntax and logic here looks good. Since you only needed this connection for one call, it makes sense to just leave it here. As soon as you make another call, I imagine you'd pull the connection and the formatting into another method. In the final, I'd like to see that abstraction, even if it is just one call that the service will be responsible for making.